### ISSUE TITLE
Ligature rendering is buggy

### ISSUE THREAD

**Issue by lassipulkkinen** [termux/termux-app#1445]
**Problem description**

When using the FiraCode font from Termux:Styling, some ligatures either don't appear or show the wrong glyph.

**Steps to reproduce**

1. Open a Termux session with Termux:Styling installed.
2. Set the font to FiraCode.
3. Type `...`.

**Expected behavior**

The ligature for `...`, with the dots shifted slightly closer together, should appear. Currently shows ` ?.` instead.

**Additional information**

`termux-info` output:

```
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://dl.bintray.com/termux/termux-packages-24/ stable main
# science-repo (sources.list.d/science.list)
deb https://dl.bintray.com/grimler/science-packages-24 science stable
# game-repo (sources.list.d/game.list)
deb https://dl.bintray.com/grimler/game-packages-24 games stable
# unstable-repo (sources.list.d/unstable.list)
deb https://dl.bintray.com/xeffyr/unstable-packages unstable main
# x11-repo (sources.list.d/x11.list)
deb https://dl.bintray.com/xeffyr/x11-packages x11 main
Updatable packages:
git/stable 2.25.0-2 aarch64 [upgradable from: 2.25.0]
Android version:
9
Kernel build information:
Linux localhost 3.18.120-perf #1 SMP PREEMPT Fri Dec 27 13:54:26 CST 2019 aarch64 Android
Device manufacturer:
HMD Global
Device model:
TA-1021
```

**Comment by xeffyr** [comment]
Font rendering is performed by Android OS. Termux does not implement font rendering on its own.

**Comment by lassipulkkinen** [comment]
I tried to use Fira Code (from Google Fonts) in another app (an image editor) to see if this really affects Android font rendering as a whole, but it worked with no issues. So I did the same with a FiraCode.otf directly from the termux-styling repo, and the font file itself turned out to be broken.

After some further research it turned out that the Fira[ Code] fonts in termux-styling are out of date. The fix was straightforward so I submitted a PR on the termux-style
